src/test/java/core/basesyntax/service/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.FruitTransactionParserImpl;
+import core.basesyntax.service.impl.OperationValidatorImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    private static List<String> testData;
+    private static FruitTransactionParser fruitTransactionParser;
+
+    @BeforeClass
+    public static void setUp() {
+        testData = new ArrayList<>();
+        OperationValidator operationValidator = new OperationValidatorImpl();
+        fruitTransactionParser = new FruitTransactionParserImpl(operationValidator);
+    }
+
+    @Before
+    public void beforeAll() {
+        testData.clear();
+        testData.add("type,fruit,quantity");
+    }
+
+    @Test
+    public void parseTransaction_getFruit_ok() {
+        testData.add("b,banana,20");
+        List<FruitTransaction> expected = new ArrayList<>();
+        expected.add(new FruitTransaction("b", new Fruit("banana"), 20));
+        List<FruitTransaction> actual = fruitTransactionParser.parseFruitTransactions(testData);
+        Assert.assertEquals(expected.get(0).getFruit(), actual.get(0).getFruit());
+        Assert.assertEquals(expected.get(0).getAmount(), actual.get(0).getAmount());
+        Assert.assertEquals(expected.get(0).getOperation(), actual.get(0).getOperation());
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void parseTransaction_null_notOk() {
+        fruitTransactionParser.parseFruitTransactions(null);
+    }
+
+    @Test
+    public void parsTransaction_wrongFruit_notOk() {
+        testData.add("b,error,0");
+        List<FruitTransaction> expected = new ArrayList<>();
+        expected.add(new FruitTransaction("b", new Fruit("banana"), 20));
+        List<FruitTransaction> actual = fruitTransactionParser.parseFruitTransactions(testData);
+        Assert.assertFalse(expected.toString(), Boolean.parseBoolean(actual.toString()));

```

Comment:
```suggestion
        Assert.assertEquals(someExpectedValue, actual);

```
let's do it this way